### DISCUSSION TITLE
Carteira não mostra numeros decimais (Wallet does not show decimal numbers)

### DISCUSSION THREAD

**Discussion by waldsonfreitas** [Chia-Network/chia-blockchain#6028] [Support]
Boa noite !, minha carteira não mostra números decimais, apenas um numero exato, como configurar ? 
Good night !, my wallet does not show decimal numbers, just an exact number, how to configure?
![Capturar](https://user-images.githubusercontent.com/84819122/119582636-44fddb00-bd9b-11eb-8beb-771405dcb282.PNG)

**Comment by tiny0little** [comment]
in case of zero it won't show, once you have more than zero, it will

**Comment by BrandtH22** [comment]
@waldsonfreitas , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
